src/pages/guides/using/marketo/marketo-data-streams.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Marketo Data Streams
```

Comment:
If we're going to have a title on 2 of the pages, we should have it on the 3rd as well (Marketo User Audit Data Stream)